docs/Reference/Server.md

```diff
@@ -1278,7 +1278,7 @@ const fastify = Fastify({
      */
     bucket: function factory (parentSchemas) {
       return {
-        addSchema (inputSchema) {
+        add (inputSchema) {
```

Comment:
Give me one sec to check deeply.

Right now, I think that `add` was wrong but I need to think better the inpacts